aws/resource_aws_macie2_account_test.go

```diff
@@ -0,0 +1,248 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/macie2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"testing"
+)
+
+func TestAccAwsMacie2Account_basic(t *testing.T) {
+	var macie2Output macie2.GetMacieSessionOutput
+	resourceName := "aws_macie2_account.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckAwsMacie2AccountDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testaccawsmacieaccountconfigBasic(),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsMacie2AccountExists(resourceName, &macie2Output),
+					resource.TestCheckResourceAttrSet(resourceName, "finding_publishing_frequency"),
+					resource.TestCheckResourceAttrSet(resourceName, "status"),
+					resource.TestCheckResourceAttrSet(resourceName, "updated_at"),
+					resource.TestCheckResourceAttrSet(resourceName, "service_role"),
+					resource.TestCheckResourceAttrSet(resourceName, "created_at"),
+					resource.TestCheckResourceAttrSet(resourceName, "updated_at"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
```

Comment:
This resource should have a "disappears" test, as described at https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/running-and-writing-acceptance-tests.md#disappears-acceptance-tests